lib/pronto/git/repository.rb

```diff
@@ -16,6 +16,7 @@ def diff(commit, options = nil)
                           else
                             merge_base = merge_base(commit)
                             patches = @repo.diff(merge_base, head, options)
+                            patches.find_similar!(:renames => true)
```

Comment:
I think this might read better expressed in the JSON format, thoughts?

```suggestion
                            patches.find_similar!(renames: true)
```

Reply:
Definitely, that conforms to the new Ruby style guide. Changed it.